src/main/resources/application.yaml

```diff
@@ -34,26 +34,25 @@ spring:
             response-type: code
             response-mode: form_post
             prompt: login
-            issuer-uri: 'https://hmctsdartsb2csbox.b2clogin.com/${spring.security.oauth2.client.registration.external-azure-ad.tenant-id}/v2.0/'
+            issuer-uri: "https://hmctsdartsb2csbox.b2clogin.com/${spring.security.oauth2.client.registration.external-azure-ad.tenant-id}/v2.0/"
             provider: external-azure-ad-provider
         provider:
           external-azure-ad-provider:
             authorization-uri: https://hmctsdartsb2csbox.b2clogin.com/hmctsdartsb2csbox.onmicrosoft.com/B2C_1_darts_externaluser_signin/oauth2/v2.0/authorize
             token-uri: https://hmctsdartsb2csbox.b2clogin.com/hmctsdartsb2csbox.onmicrosoft.com/B2C_1_darts_externaluser_signin/oauth2/v2.0/token
             jwk-set-uri: https://hmctsdartsb2csbox.b2clogin.com/hmctsdartsb2csbox.onmicrosoft.com/B2C_1_darts_externaluser_signin/discovery/v2.0/keys
 
-
   datasource:
     driver-class-name: org.postgresql.Driver
     url: jdbc:postgresql://${DARTS_API_DB_HOST:localhost}:5432/${DARTS_API_DB_NAME:darts}${DARTS_API_DB_OPTIONS:}
-    username: ${DARTS_API_DB_USERNAME:darts}
-    password: ${DARTS_API_DB_PASSWORD:darts}
+    username: ${DARTS_API_DB_USERNAME:postgres}
```

Comment:
having this wrong was useful to prove that environment variables were not being imported correctly as we could see it defaulting to darts, but suppose that shouldn't happen again :-\

Reply:
Changed back.